Array/Largest_sum_subarray.cpp

```diff
@@ -0,0 +1,55 @@
+/* Given an array of positive and negative integers. We have to find subarray having maximum sum */
+#include<iostream>
+using namespace std;
+
+// Function to find the maximum sum of a contiguous subarray
+int kadaneAlgo(int arr[], int n)
+{
+    // stores the maximum sum subarray found so far
+    int max_so_far = INT_MIN;
```

Comment:
‘INT_MIN’ was not declared in this scope. Include limits.h header file to resolve this error.

Reply:
Recheck please, updated the code